modules/globebrowsing/scripts/layer_support.lua

```diff
@@ -8,6 +8,55 @@ openspace.globebrowsing.documentation = {
         Name = "createHeightLayers",
         Arguments = "table",
         Documentation = "Creates a table used in the 'HeightLayers' of a RenderableGlobe."
+    },
+    {
+        Name = "createTemporalGibsGdalXml",
```

Comment:
Do you think it is ok that it takes these many arguments (layerName, startDate, endDate, timeResolution, resolution, format) or should it take a dictionary with key-value pairs instead?